experimental/kubernetes-v2/config/clouddriver.yml

```diff
@@ -0,0 +1,92 @@
+server:
```

Comment:
Feels like it may end up being a bit of a maintenance headache to try to synchronize changes from the base files to these files.

If the bulk of the contents of the files are unchanged, is there a way to derive these from the base files during the script execution?